src/algebra/binary-exp.md

```diff
@@ -67,4 +67,6 @@ $a^{b} = (a^{b/2})^2 \quad$ for even `b`,
 $a^{b} = (a^{(b-1)/2})^2 \cdot a \quad$ for odd `b`,  
 $a^{1} = a$.
 
-
+Practice problems from UVA OJ:
```

Comment:
Please use the same format for this section as in https://github.com/e-maxx-eng/e-maxx-eng/blob/master/src/data_structures/fenwick.md :
* title "Practice Problems"
* unordered list of problems
* each problem title is a link to a page of this problem on an OJ

Please also apply this format to the other pull requests.